dyn_em/start_em.F

```diff
@@ -866,6 +869,8 @@ SUBROUTINE start_domain_em ( grid, allowed_to_read &
       DO k = kts,kte
          DO i = its, min(ite,ide-1)
             z_at_q(i,k,j)=(grid%ph_2(i,k,j)+grid%phb(i,k,j))/g
+            PRES(i,k,j) = grid%p(i,k,j)+grid%pb(i,k,j)              ! ntu3m
```

Comment:
@tzuchin12 
Here also, and I am assuming below. Are these required if NTU is not the MP chosen?

Reply:
They are required only for the NTU scheme.